bridge/bridge.go

```diff
@@ -11,8 +11,15 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	dockerapi "github.com/fsouza/go-dockerclient"
+	"github.com/rancher/go-rancher-metadata/metadata"
+)
+
+const (
+	metadataURL = "http://rancher-metadata/2015-12-19" 
```

Comment:
Any reason for the the decision to use `2015-12-19` rather than a newer `2016-07-29` which is considered `latest` in Rancher v1.6.2?